jina/drivers/__init__.py

```diff
@@ -242,10 +231,10 @@ def __init__(self, depth_range: Tuple[int] = (0, 1), apply_order: str = 'post',
         if isinstance(traverse_on, str):
             traverse_on = (traverse_on,)
         self.traverse_fields = set(traverse_on)
-        if apply_order in {'post', 'pre'}:
```

Comment:
right now `pre` is not used anywhere and it just adds noise to the code, it is easy to add when required